### ISSUE TITLE
Ability Cost + modelling

### ISSUE THREAD

**Issue by davebytemfg** [Jon-Davis/Cypher-System-JSON-DB#15]
I'm not advocating for this, but wanted to mention it for completeness.

Currently the cost is modeled as a integer, losing the + indicator of this behavior in the ability.

Option A: Make it a string?
Option B: Add some other flag indicating additional costs are possible? (preferred?)

Right now I can't think of what I'd do with this information other than display it, but pedantically it might be a thing to think about.
The information is in the description for the player to process, I think the + is redundant with that.

from the SRD:
_Sometimes the point cost for an ability has a + sign after the number. For example, the cost might be given as “2+ Intellect points.” That means you can spend more points or more levels of Effort to improve the ability further, as explained in the ability description._

Some examples:
_Disguise Other (4+ Intellect points): You apply your shapechanging ability to another creature of your size or smaller, giving them a form that you are able to assume. This lasts for about ten minutes. In addition to the normal options for using Effort, you can choose to use Effort to increase the duration; one level of Effort increases it to an hour, two increases it to a day. A creature can revert to its normal form as an action, but it cannot then change back into the altered form. You probably can't use Disguise Other to disguise a kind of creature that is very different from you, such as a human disguising a robot, animal, or crystalline alien. Action._

_Disrupting Touch (1+ Might points): You can turn your Phase Sprint into a melee attack by purposefully grazing another creature as you run. When you do, the touch releases a violent blast of energy that inflicts 2 points of damage to the target (ignores Armor). Whether you hit or miss, your movement (and turn) ends immediately, which puts you within immediate distance of your target. If you apply Effort to increase damage rather than to ease the task, you deal 2 additional points of damage per level of Effort (instead of 3 points); the target takes 1 point of damage even if you fail the attack roll. Enabler._

**Comment by Jon-Davis** [comment]
Yeah this is one of those issues that I wanted to get to at some point. 

I decided to go with both options.
1. I added a "cost_rendered" field to each ability. I figure it will be common for anyone using the file to want to print abilities similar to how they appear in the text, so I might as well save everyone the work of reverse processing my split process 😁
2. I added a "+" to the additional_cost field if the ability can utilize additional points in it's pool.

This way both use cases should be easy to use.
